src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -438,31 +416,36 @@ private static PageDescriptionEntity GetPageInfo(Page page, string pagesInfoPath
         /// <param name="documentGuid">Document guid.</param>
         /// <param name="settings">Viewer settings object.</param>
         /// <returns>Page content as a string.</returns>
-        private static string GetPageContent(Page page, string password, string documentGuid, ViewerSettings settings)
+        private static string GetPageContent(int pageNumber, string password, string documentGuid, string cachePath, int passedPageNumber = -1, int newAngle = 0)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'password'.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5933705)